src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+
+    },
+    removeBook: () => {
+
```

Comment:
- [ ] Please create the  2 reducers, `addBook` and `removeBook` here, this is part of the requirements, See suggestion 👍🏿 
```suggestion
  ```javascript
 addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
```

```